src/SFML/Audio/SoundFileWriterOgg.cpp

```diff
@@ -63,7 +63,8 @@ bool SoundFileWriterOgg::open(const std::filesystem::path& filename, unsigned in
     m_channelCount = channelCount;
 
     // Initialize the ogg/vorbis stream
-    ogg_stream_init(&m_ogg, std::rand());
+    static std::mt19937 rng(std::random_device{}());
+    ogg_stream_init(&m_ogg, std::uniform_int_distribution(0, RAND_MAX)(rng));
```

Comment:
Feels a bit odd to use `RAND_MAX` here, which is essentially more connected to `std::rand()`. It also means, that technically, you wouldn't be allowed to remove the `cstdlib` header.

Can't we just use int limit or some other large number?

Reply:
I mean one could argue whether we should actually use some unique identifier generator, but I guess a random number is okay for a short lived writing process. To prevent potential collisions, you'd just want a big pool for the distribution, so I'd just take `std::numeric_limits<int>::max()`.